code/modules/client/preference/preferences.dm

```diff
@@ -1308,7 +1308,7 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 					else //Not using the whitelist? Aliens for everyone!
 						new_species += GLOB.whitelisted_species
 
-					species = input("Please select a species", "Character Generation", null) in new_species
+					species = input("Please select a species", "Character Generation", null) in sortTim(new_species, /proc/cmp_text_asc)
```

Comment:
I think you can just use sortList() instead of these. Same effect with less overhead

Reply:
I can, but sortTim(uses Timsort) is more optimized compared to sortList(Mergesort)